packages/components/src/LightBox/LightBox.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import ExternalLightBox from "react-image-lightbox";
+import styles from "./LightBox.css";
+
+// Library requires fetching its CSS.
+// eslint-disable-next-line import/no-internal-modules
+import "react-image-lightbox/style.css";
+
+interface PresentedImage {
+  title?: string;
+  caption?: string;
+  url: string;
+}
+
+interface LightBoxProps {
+  readonly open: boolean;
+  readonly images: PresentedImage[];
+  readonly imageIndex?: number;
+  onRequestClose(): void;
+}
+
+export function LightBox({
+  open = false,
+  images,
+  imageIndex = 0,
+  onRequestClose,
+}: LightBoxProps) {
+  const [currentImageIndex, setCurrentImageIndex] = useState(imageIndex);
+  let nextSrc = undefined;
+  let prevSrc = undefined;
+  if (images.length > 1) {
+    nextSrc = images[(currentImageIndex + 1) % images.length].url;
+    prevSrc =
+      images[(currentImageIndex + images.length - 1) % images.length].url;
+  }
+
+  return (
+    <>
+      {open && (
+        <ExternalLightBox
+          wrapperClassName={styles.wrapper}
+          mainSrc={images[currentImageIndex].url}
+          enableZoom={false}
+          nextSrc={nextSrc}
+          prevSrc={prevSrc}
+          imageTitle={images[currentImageIndex].title}
+          imageCaption={images[currentImageIndex].caption}
+          onCloseRequest={onRequestClose}
+          onMovePrevRequest={() =>
+            setCurrentImageIndex(
+              (currentImageIndex + images.length - 1) % images.length,
+            )
+          }
```

Comment:
It's generally not great to declare handlers inline like this. Would probably be better to declare a `function handleMovePrevious()` after the return value and then pass it in here instead.

```suggestion
          onMovePrevRequest={handleMovePrevious}
```

When you inline handlers causes them to be alloced and destroyed with every component render which can have a performance hit.